### DISCUSSION TITLE
Tooltip containing a link - delay tooltip exit until mouse reaches tooltip content

### DISCUSSION THREAD

**Discussion by uriklar** [radix-ui/primitives#1690] [Help]
Hi! I'm using radix-ui/react-tooltip as a basis for our design system tooltip component (replacing an existing implementation), and we currently have a prop called `delayTooltipExit` which is used for tooltips that have a link inside them. 
The use case is that we want to keep the tooltip from disappearing while the user is traveling with their mouse to click the link.
I was wondering if anyone here implemented such a use case? Is it even legit a11y-wise to put a link in a tooltip? 
Thanks!

**Comment by benoitgrelard** [comment]
> Is it even legit a11y-wise to put a link in a tooltip?

It isn't. Tooltip should not contain any interactive elements.

Regardless though, we handle moving the mouse from the trigger to the content so you can select the text for example.
